golos.charge/golos.charge.cpp

```diff
@@ -61,28 +61,25 @@ void charge::use(name user, symbol_code token_code, uint8_t charge_id, int64_t p
     consume_charge(issuer, user, token_code, charge_id, price_arg, cutoff_arg, vesting_price);
 }
 
-void charge::useandstore(name user, symbol_code token_code, uint8_t charge_id, int64_t stamp_id, int64_t price_arg) {
-    auto issuer = token::get_issuer(config::token_name, token_code);
-    require_auth(issuer);
-    auto new_val = consume_charge(issuer, user, token_code, charge_id, price_arg);
-    
-    storedvals storedvals_table(_self, user.value);
-    auto storedvals_index = storedvals_table.get_index<"symbolstamp"_n>();
-    auto k = stored::get_key(token_code, charge_id, stamp_id);
-    auto itr = storedvals_index.find(k);
-    if (itr != storedvals_index.end())
-        storedvals_index.modify(itr, issuer, [&]( auto &item ) { item.value = new_val.data(); });
-    else
-        storedvals_table.emplace(issuer, [&]( auto &item ) {
-            item.id = storedvals_table.available_primary_key();
-            item.symbol_stamp = k;
-            item.value = new_val.data();
-        });
-
-    INLINE_ACTION_SENDER(golos::publication, calcrwrdwt) (config::publish_name,
-        {_self, config::active_name},
-        {user, token_code, stamp_id, new_val.data()});
-}
+// this function is not used
```

Comment:
Please, do not change this function. This function can be used in another places. In any case this function will be removed in another issue.